src/ui/public/index_patterns/index_patterns_api_client.js

```diff
@@ -0,0 +1,102 @@
+import { resolve as resolveUrl, format as formatUrl } from 'url';
+
+import { pick, mapValues } from 'lodash';
+import 'whatwg-fetch';
```

Comment:
It isn't, removing. I was originally planning to use it, but using the `$http` service helps this client integrate better with existing code.